content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md

```diff
@@ -0,0 +1,59 @@
+---
+date: '2023-08-09'
+version: 'Corda 5.1'
+title: "Gateway Worker"
+version: 'Corda 5.1'
+menu:
+  corda5:
+    parent: corda5-cluster-metrics
+    identifier: corda5-cluster-gateway-worker
+    weight: 13000
+section_menu: corda5
+---
+
+# Gateway Worker
+
+The P2P {{< tooltip >}}gateway worker{{< /tooltip >}} is tasked with sending outbound messages to virtual nodes located in
+different clusters and receiving inbound messages from virtual nodes within different clusters. To accomplish this,
+a gateway worker establishes TLS connections with gateway workers in other clusters and exchanges messages through HTTPS requests.
+
+In the outbound direction, internal components within the cluster forward messages intended for the gateway worker through a
+message bus (for example, Kafka). Similarly, in the inbound direction, the gateway worker forwards any messages received via HTTPS
+from other clusters to the local cluster through the message bus.
+
+The P2P gateway worker is additionally tasked with conducting certificate revocation checks. These checks are carried out
+either as part of the TLS handshake with other gateway workers or on behalf of other components within the internal network zone of
+the cluster that cannot reach out to the public Internet. The gateway worker receives requests to perform these revocation
+checks and returns the results via a message bus.
+
+You can observe the behavior of these functions using the following metrics: rate or latency of inbound/outbound requests,
+number of inbound/outbound TLS connections, and rate or latency of revocation checks.
+
+<style>
+table th:first-of-type {
+    width: 25%;
+}
+table th:nth-of-type(2) {
+    width: 10%;
+}
+table th:nth-of-type(3) {
+    width: 20%;
+}
+table th:nth-of-type(4) {
+    width: 45%;
+}
+</style>
+
+| Metric | Type | Tags | Description |
+| :----------- | :----------- | :----------- | :----------- |
+| `corda_p2p_gateway_inbound_request_time_seconds` | Timer | <ul><li>`response.type`</li><li>`endpoint.source`</li></ul> | The latency and the number of requests from a gateway worker to peer gateway workers in other clusters. |
+| `corda_p2p_gateway_outbound_request_time_seconds` | Timer | <ul><li>`response.type`</li><li>`endpoint.destination`</li></ul> | The number of incoming requests from peer gateway workers in other clusters and the time it took to process them. Main way of processing on the inbound requests is identifying the partition where a session is hosted and writing the message to Kafka, so that it’s later processed by the appropriate link manager. |
```

Comment:
`Main way of processing` doesn't sound right. In the original doc, it was `main path of processing` which refers to what's executed as part of the processing.

But, I think we can just remove this sentence altogether as it's quite low-level detail for an external audience that's not present in other metrics.

Reply:
Removed the sentence.